src/solidity-frontend/solidity_convert.cpp

```diff
@@ -1914,6 +1928,40 @@ bool solidity_convertert::get_unbound_expr(
   // it's not a member access, as it can only jump within current contract
   std::string c_name = current_contractName;
   assert(!c_name.empty());
+  if (expr.contains("expression") &&
+  expr["expression"]["nodeType"] == "MemberAccess")
+  {
+  const auto &member_access = expr["expression"];
+  const auto &base_expr = member_access["expression"];
+
+    if (base_expr["nodeType"] == "Identifier" &&
+        base_expr.contains("referencedDeclaration"))
+    {
+      int ref_id = base_expr["referencedDeclaration"];
+      const auto &ref_decl = find_decl_ref(ref_id);
+
+      if (ref_decl["nodeType"] == "ContractDefinition" &&
+          ref_decl["contractKind"] == "library")
+      {
+        std::string lib_name = ref_decl["name"];
+        std::string instance_name, instance_id;
+        get_static_contract_instance_name(lib_name, instance_name, instance_id);
+
+        const symbolt *lib_sym = context.find_symbol(instance_id);
+
+        exprt lib_expr = symbol_expr(*lib_sym);
+        std::string func_name = member_access["memberName"];
+
+        code_function_callt func_call;
+        locationt l;
+        get_location_from_node(expr, l);
+        func_call.location() = l;
+        func_call.function() = member_exprt(lib_expr, func_name, code_typet{});
```

Comment:
I am not sure if we need to consider the unbound? maybe check how SMTChecker perform. If in unbound, SMTChecker models it as arbitrary external call, then we need to do the same.